# Calibration file for the 3DM-GX4-25
# Mounted on IRIS w/ DJIS800

# Noise parameters of sensors
noise_std:
  accel:
    x: 1.0
    y: 1.0
    z: 1.0
  gyro:
    x: 0.001
    y: 0.001
    z: 0.001
  mag:
    x: 0.3
    y: 0.3
    z: 0.3

# Threshold used to enable bias calculation
gyro_bias_thresh: 0.01
